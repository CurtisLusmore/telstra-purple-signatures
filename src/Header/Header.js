import React from 'react';
import './Header.scss';

const Header = ({ brandInfo }) => (
  <nav className="navbar" role="navigation" aria-label="main navigation">
    <div className="section" style={{ width: '100%' }}>
      <div className="container">
        <div className="navbar-brand">
          <a className="navbar-item" href={brandInfo.brandLink}>
            {brandInfo.appName}
          </a>
        </div>
      </div>
      <div className="column" />
    </div>
  </nav>
);

export default Header;
