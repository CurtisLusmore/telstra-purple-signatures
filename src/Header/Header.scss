.navbar {
  background-color: #141b33;
  color: white;

  .navbar-brand {
    background: url(https://labs.readify.net/Content/readify-device.png) no-repeat 15px 50%;
    padding-left: 50px;

    a {
      color: white;
      font-size: 26px;
      font-weight: 200;
    }
  }
}