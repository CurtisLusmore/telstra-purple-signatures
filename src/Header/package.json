{
  "main": "./Header.js",
  "name": "Header",
  "version": "0.0.0",
  "private": true
}