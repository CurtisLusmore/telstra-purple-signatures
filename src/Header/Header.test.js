import React from 'react';
import { shallow } from 'enzyme';

import Header from './Header';

it('Renders', () => {
  const props = {
    brandInfo: {
      brandLink: 'https://brand-link',
      appName: 'Test App'
    }
  };
  const tree = shallow(<Header {...props} />);
  expect(tree).toMatchSnapshot();
});
