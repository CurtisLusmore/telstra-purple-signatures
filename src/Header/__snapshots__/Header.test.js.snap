// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<nav
  aria-label="main navigation"
  className="navbar"
  role="navigation"
>
  <div
    className="navbar-brand"
  >
    <a
      className="navbar-item"
      href="https://brand-link"
    >
      Test App
    </a>
  </div>
</nav>
`;
